### ISSUE TITLE
Password AutoFill support (iOS 12)

### ISSUE THREAD

**Issue by ghost** [MiniKeePass/MiniKeePass#613]
Announced recently is the new password manager API in iOS 12:
https://developer.apple.com/documentation/authenticationservices?language=objc

It would be great if we could have keepass as a provider!

**Comment by abcdev** [comment]
WWDC session:[https://developer.apple.com/videos/play/wwdc2018/721/]( https://developer.apple.com/videos/play/wwdc2018/721/)

**Comment by Nubzori** [comment]
iOS 12 releases tomorrow, this would be a very welcome feature!

**Comment by shackrock** [comment]
yes PLEASE!  I am almost ready to switch to LastPass now that this is added to iOS12, but much rather stick w/ Keepass!

**Comment by timotheeeee** [comment]
I will stick with MiniKeePass no matter what, however this feature would be awesome!

**Comment by JLWFuQrioea69ugsykvQcg** [comment]
This project has been dead for a while now and I don’t see that changing anytime soon.

**Comment by BastianLinge** [comment]
If this isnt implemented soon, Im changing to another App.

**Comment by kgraefe** [comment]
Do you  know a Keepass app that has autofill support?

**Comment by SiGamma** [comment]
I don't know of any actually open source KeePass app besides this one on iOS, period, autofill support or not. 

I was considering diving in and implementing AutoFill support myself, but considering devs are completely silent and aren't even approving current pull requests (such as those adding KDBX4 support), I kinda gave up. 

Only solution I can see is forking this project and getting another app on the App Store.

**Comment by milesmcclane** [comment]
I would pay for a decent none dead version of this if you actually would consider forking it yourself. I’m sure many would, so it would be worth your while to try if you have the time!

**Comment by deucalion** [comment]
Maybe they lost track of time, reviewing the pull requests (e.g. with #492 ).

I'd be happy to donate to both you @SiGamma and the reviewers / developers (@tssmith7 and @jarush ?) if you care to make a PR and them to review it.

I'm not sure if a fork is the best solution.

_(There already is a fork with kdbx4 at https://github.com/abcdev/MiniKeePass )_

**Comment by BastianLinge** [comment]
Hello,
found a iOS App with autofill support for Keepass. KyPass 4.4
https://www.kyuran.be/blog/2018/09/18/kypass-v4-4-0-is-now-available/

**Comment by themultiplexer** [comment]
I forked it myself and it is in proof of concept stage now but works for me just fine. Hardcoded the database file as "keeps.kdbx" and copyied it to the extension. 

![Demo](https://user-images.githubusercontent.com/22082747/51275575-a72bc180-19d2-11e9-8e8a-bb60b0b0d33b.gif)

**Comment by metbril** [comment]
This feature has been implemented recently in KeePass Touch. This is a fork of the badly maintained MiniKeePass.
